rust-analyzer/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM alpine as build
+
+WORKDIR /build
+
+RUN apk add --no-cache curl
+
+RUN curl -L https://github.com/rust-analyzer/rust-analyzer/releases/latest/download/rust-analyzer-linux -o /build/rust-analyzer \

```

Comment:
We'd like everything to be statically versioned when possible. Could you please add a version ENV and download based on that rather than downloading latest?

Reply:
No problem.